src/cards/ha-entities-card.html

```diff
@@ -129,6 +135,15 @@
 
     return canToggleCount > 1;
   }
+
+  ready() {
+    //super.ready();
+    var CardsWithMore = this.$.querySelectorAll('.state .has-more-info-card');
```

Comment:
I think this.$ is a map to unconditional children with IDs.
Try `this.querySelectorAll`

Reply:
CardsWithMore.length is also 0